src/BinaryParsers/ElfBinary/Dwarf/DwarfCompilationUnit.cs

```diff
@@ -263,7 +264,27 @@ private void ReadData(IDwarfBinary dwarfBinary, DwarfMemoryReader debugData, Dwa
                         case DwarfFormat.Strx:
                         case DwarfFormat.GNUStrIndex:
                             attributeValue.Type = DwarfAttributeValueType.String;
-                            attributeValue.Value = debugData.LEB128() + (ulong)beginPosition;
+                            debugData.LEB128();
+                            break;
+
+                        case DwarfFormat.Strx1:
```

Comment:
>Strx1 [](http://example.com/codeflow?start=41&length=5)

These format types currently exist in some test binaries but we're not handling them properly. For now, I just seek forward into the debug data streams so that follow on attributes don't get corrupted. In a next change, we will retrieve the actual values of these strings.